src/app/open-api-generator/parser/open-api-parser.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * responsible for parsing the OpenAPI Document/Swagger Document
+ * and replacing all occurrences of ${multi-line-comment-end} substring
+ * with `*'/`
+ */
+
+import * as logger from "../../../util/logger";
+
+type TypeSchema = {
+  type?: string;
+  description?: string;
+  properties?: Record<string, TypeSchema>;
+  example?: string;
+};
+
+export function fixTypesDescriptions(
+  type: TypeSchema,
+  schemaType: string,
+  name?: string,
+) {
+  if (!type) {
+    return;
+  }
+
+  const visited = new Set<string>();
+  traverseTypes(name ? name : `${schemaType}-${name}`, type, visited);
+
+  // const typedTypes = types as Record<string, TypeSchema>;
+  // for (const currentType in typedTypes) {
+  //   const visited = new Set<string>();
+  //   traverseTypes(currentType, typedTypes[currentType]!, visited);
+  // }
```

Comment:
nit: stray comment

Reply:
Removed in [551aced](https://github.com/hasura/ndc-open-api-lambda/pull/21/commits/551aced67e725f9131f1872e8927e79ccd2a290b)